### DISCUSSION TITLE
Paste Text From Clipboard

### DISCUSSION THREAD

**Discussion by jwoodmanae** [AutomaApp/automa#1213] [Q&A]
Any way to paste text from the clipboard? Trying to transfer data from a spreadsheet into a field within a different tab one at a time. The clipboard has a copy function but no paste. I've tried JavaScript but nothing has worked.

**Comment by fmen1** [comment]
Not true, Clipboard has the "Get Clipboard Data" as well as the "Insert Text to Clipboard" functions. Check it out.

**Reply by SuperTempAccount** [reply]
You won't be able to paste, but you can copy the data from the clipboard to a table or variable then use the form block to write it into the new location.
Just note that you'll lose formatting in the process. It seems that when you assign the clipboard data to a variable, it only retains the text value.

Edit: turns out there is a limitation with the browser for security reasons. Automa can access only plain text in the clipboard. Not rich formatted text or other objects like images.
